### ISSUE TITLE
[FR] Would like to support vmess+tcp+http masquerading

### ISSUE THREAD

**Issue by suvonyang** [hiddify/hiddify-app#794]
### Feature description

The current version does not support vmess+tcp+http masquerading, hopefully the subsequent versions will support it!

**Comment by UbiVPN** [comment]
It feels like there's not much point in disguising HTTP anymore, considering there are so few purely HTTP websites nowadays. Maybe look into TLS instead.

**Comment by suvonyang** [comment]
Yes. Probably because that's what it is, the combo was used for months without being blocked by firewalls. Before that, both tls, and reality, were blocked by firewalls in short order.
